app/controllers/bookmarks_controller.rb

```diff
@@ -9,7 +9,9 @@ def index
 
   def new
     @bookmark            = Bookmark.new
-    @bookmark.name       = params[:query].to_s.strip.split(/\s| = |!|~|>|</)[0]
+    query                = params[:query].to_s.squish
```

Comment:
There is a corner case, where I would want to search for double-spaces in exact search `name = "Some  very bad name"`. It's a corner case but still, maybe we should rather handle the autocompleter behaviour to not add the additional spaces in the first place: it would look a bit better. Another reason for separate PR on that.